packages/drivers/routerlicious-driver/src/documentService.ts

```diff
@@ -57,34 +53,14 @@ export class DocumentService implements api.IDocumentService {
             this.tokenProvider,
             this.logger,
             this.gitUrl,
-            this.directCredentials,
         );
         const historian = new Historian(
             this.gitUrl,
-            this.historianApi,
-            this.disableCache,
+            true,
+            false,
             storageRestWrapper);
         const gitManager = new GitManager(historian);
 
-        // Insert cached seed data
-        if (this.gitCache !== undefined) {
-            for (const ref of Object.keys(this.gitCache.refs)) {
-                gitManager.addRef(ref, this.gitCache.refs[ref]);
-            }
-
-            for (const commit of this.gitCache.commits) {
-                gitManager.addCommit(commit);
-            }
-
-            for (const tree of this.gitCache.trees) {
-                gitManager.addTree(tree);
-            }
-
-            for (const blob of this.gitCache.blobs) {
-                gitManager.addBlob(blob);
-            }
-        }
-
```

Comment:
This is actually a useful capability although no one uses it today. This parameter actually lets the host seed the driver with summary data which means when client calls getLatestSummary(), the data is already cached locally.

I am okay with removing it now but can you open an internal issue to bring it back when we start onboarding hosts.

Reply:
Sounds good. When we do add it back, I'd be interested to work with you to figure out a way to make this useful functionality more obviously useful. Perhaps we can add some examples in the r11s-driver README about when this is good/possible to take advantage of.